stable/prometheus-adapter/templates/hpa-external-metrics-cluster-role-binding.yaml

```diff
@@ -1,5 +1,5 @@
-
 {{- if .Values.rbac.create -}}
```

Comment:
Combine these two if statements with `and`

Reply:
Done.